src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo,
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {todo.title}
+      </h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
Don't render `UserInfo` component if there is no user. It's redundant 
```suggestion
     {todo.user && <UserInfo user={todo.user} />}
```